src/main/java/org/xbill/DNS/Type.java

```diff
@@ -9,181 +9,184 @@
  *
  * @author Brian Wellington
  */
-
 public final class Type {
 
-/** Address */
+/** {@link ARecord Address} */
 public static final int A		= 1;
 
-/** Name server */
+/** {@link NSRecord Name server} */
 public static final int NS		= 2;
 
-/** Mail destination */
+/** {@link MDRecord Mail destination} */
 public static final int MD		= 3;
 
-/** Mail forwarder */
+/** {@link MFRecord Mail forwarder} */
 public static final int MF		= 4;
 
-/** Canonical name (alias) */
+/** {@link CNAMERecord Canonical name (alias)} */
 public static final int CNAME		= 5;
 
-/** Start of authority */
+/** {@link SOARecord Start of authority} */
 public static final int SOA		= 6;
 
-/** Mailbox domain name */
+/** {@link MBRecord Mailbox domain name} */
 public static final int MB		= 7;
 
-/** Mail group member */
+/** {@link MGRecord Mail group member} */
 public static final int MG		= 8;
 
-/** Mail rename name */
+/** {@link MRRecord Mail rename name} */
 public static final int MR		= 9;
 
-/** Null record */
+/** {@link NULLRecord Null record} */
 public static final int NULL		= 10;
 
-/** Well known services */
+/** {@link WKSRecord Well known services} */
 public static final int WKS		= 11;
 
-/** Domain name pointer */
+/** {@link PTRRecord Domain name pointer} */
 public static final int PTR		= 12;
 
-/** Host information */
+/** {@link HINFORecord Host information} */
 public static final int HINFO		= 13;
 
-/** Mailbox information */
+/** {@link MINFORecord Mailbox information} */
 public static final int MINFO		= 14;
 
-/** Mail routing information */
+/** {@link MXRecord Mail routing information} */
 public static final int MX		= 15;
 
-/** Text strings */
+/** {@link TXTRecord Text strings} */
 public static final int TXT		= 16;
 
-/** Responsible person */
+/** {@link RPRecord Responsible person} */
 public static final int RP		= 17;
 
-/** AFS cell database */
+/** {@link AFSDBRecord AFS cell database} */
 public static final int AFSDB		= 18;
 
-/** X.25 calling address */
+/** {@link X25Record X.25 calling address} */
 public static final int X25		= 19;
 
-/** ISDN calling address */
+/** {@link ISDNRecord ISDN calling address} */
 public static final int ISDN		= 20;
 
-/** Router */
+/** {@link RTRecord Router} */
 public static final int RT		= 21;
 
-/** NSAP address */
+/** {@link NSAPRecord NSAP address} */
 public static final int NSAP		= 22;
 
-/** Reverse NSAP address (deprecated) */
+/** {@link NSAP_PTRRecord Reverse NSAP address (deprecated)} */
```

Comment:
see also comment about Type.SPF

Reply:
NSAP_PTR.  I never could find any explicit place where it was deprecated, but I don't think anyone uses it.  Marked obsolete in NSAP_PTRRecord javadoc.